css/src/atomics/border.scss

```diff
@@ -82,6 +86,10 @@
 	border-radius: $border-radius-lg !important;
 }
 
+.border-radius-full {
```

Comment:
should this be rounded instead of `full`?

Reply:
There is a conversation about the best name for it in Jeremy's comment thread. Please check it out

Reply:
updated